packages/engine/src/tools/token-cost.ts

```
@@ -0,0 +1,167 @@
+import { parse } from 'node-html-parser'
+import { getTokenCount } from '../utils/tokenizer.js'
+import type { MCPResponse } from '../types.js'
+
+function estimateTailwindEquivalent(html: string): string {
+  const root = parse(html)
+
+  const traverse = (node: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] 'twClasses' is never reassigned. Use 'const' instead.

(prefer-const)